Buildings/Experimental/DHC/Networks/Steam/Examples/DistributionCondensatePipe.mo

```diff
@@ -0,0 +1,105 @@
+within Buildings.Experimental.DHC.Networks.Steam.Examples;
+model DistributionCondensatePipe
+  "Example model for the steam heating distribution network"
+  extends Modelica.Icons.Example;
+
+  package MediumSte = Buildings.Media.Steam "Steam vapor medium";
+  package MediumWat =
+    Buildings.Media.Specialized.Water.TemperatureDependentDensity
+    "Liquid water medium";
+
```

Comment:
`vapor` and `Liquid` in medium description can be removed if this was not intentionally written.